content/courses/program-security/owner-checks.md

```diff
@@ -227,19 +202,18 @@ pub struct AdminConfig {
 }
 ```
 
-#### Use Anchor’s `#[account(owner = <expr>)]` constraint
+### Use Anchor's `#[account(owner = <expr>)]` constraint
 
-In addition to the `Account` type, you can use an `owner` constraint. The
-`owner` constraint allows you to define the program that should own an account
-if it’s different from the currently executing one. This comes in handy if, for
-example, you are writing an instruction that expects an account to be a PDA
-derived from a different program. You can use the `seeds` and `bump` constraints
-and define the `owner` to properly derive and verify the address of the account
-passed in.
+In addition to the `Account` type, you can use the `owner` constraint to specify
```

Comment:
Add a link to https://www.anchor-lang.com/docs/account-constraints (unfortunately Anchor doesn't support, er, anchors)

Reply:
Updated anchor link as per comment.